baselines/ppo2/ppo2.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, *, policy, ob_space, ac_space, nbatch_act, nbatch_train,
         with tf.variable_scope('ppo2_model', reuse=tf.AUTO_REUSE):
             act_model = policy(nbatch_act, 1, sess)
             train_model = policy(nbatch_train, nsteps, sess)
+            eval_model = policy(1, 1, sess)
```

Comment:
why is eval_model needed?

Reply:
We have added that to evaluate the learned policy on a single instance of our environment